templates/todo/projects/csharp-sql-swa-func/.repo/bicep/infra/main.bicep

```diff
@@ -25,13 +25,13 @@ param sqlDatabaseName string = ''
 param sqlServerName string = ''
 param webServiceName string = ''
 param apimServiceName string = ''
+param appUser string = 'appUser'
+param sqlAdmin string = 'sqlAdmin'
+param connectionStringKey string = 'AZURE-SQL-CONNECTION-STRING'
 
 @description('Flag to use Azure API Management to mediate the calls between the Web frontend and the backend API')
 param useAPIM bool = false
 
-@description('API Management SKU to use if APIM is enabled')
```

Comment:
Why removed?

Reply:
Because the previous version of `api-management` AVM module does not support SKU Type as `Consumption`,
Now the latest version supports SKU Type as `Consumption`, so I have updated the code to add `apimSku` parameter.

Reply:
Can you give me an example or pointer to that?

Reply:
When `apimSku = 'Consumption'`, the `skuCount` is required to be set to `0`, but in previous versions of AVM, `skuCount` can only be set to `1` or `2`. The relevant issue is here: https://github.com/Azure/bicep-registry-modules/issues/1628.

This issue has been fixed now, and I have also changed the relevant code: https://github.com/Azure/azure-dev/pull/3976/commits/3163c23b50e6402cb370b2e545271b4791172df4#diff-d45a036db1c971e6ad5518298f2c7e991e1e932616c6101d92eb0c1d4522e2a7R36

Reply:
Thanks